documentation/SolutionsRegistryFormat.md

```diff
@@ -49,6 +49,12 @@ The `settingsHandlers` block is unique and one of the most important blocks in t
         "options": {
             "filename": "${{environment}.APPDATA}\\Texthelp Systems\\RWSettings10.xml"
         },
+        "supportedSettings": [
```

Comment:
As per our discussion today on GPII-228, we should avoid structuring this as an array (which is a bad approach in general) but instead as a hash with a "suitably rich right-hand-side" - with room for expansion as we discover more metadata we might need to store about the settings. As a suggestion, there should at least be room for a member named "schema" holding a JSON schema chunk for the setting, which as we discovered earlier will also be able to incorporate a default value.